runtime-linux.md

```diff
@@ -1,8 +1,18 @@
 # Linux Runtime
 
 ## File descriptors
+
 By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
+The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
+Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
+
+## Dev symbolic links
 
-The runtime may pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
+After the container has `/proc` mounted, the following standard symlinks MUST be setup within `/dev/` for the io.
 
-Some of the file descriptors may be redirected to `/dev/null` even though they are open.
+|    Source       | Destination |
```

Comment:
[ln(1)][1] calls these “target” and “link name” respectively.

[1]: http://www.man7.org/linux/man-pages/man1/ln.1.html